### PR TITLE
Adds usage instructions 

### PR DESCRIPTION
**Pull request by vaporwavie**
I took [https://github.com/mgonto/restangular]() code example's in order to adapt to this project. Also added a step-by-step guide on how to use ng-restangular to any other project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+
+# Configuring Ng-Restangular
+After configuring the whole environment, it's simple to start using Ng-Restangular. Inject this snippet into your Controller.
+````typescript
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+import { RestangularModule } from 'ng-restangular';
+
+// Function for settting the default restangular configuration
+export function RestangularConfigFactory (RestangularProvider) {
```
Comment (RodolfoSilva) [inline README.md:19]: The `RestangularProvider` should be `config: RestangularConfig`.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+  RestangularProvider.setBaseUrl('http://api.youraddress.local/v1');
```
Comment (RodolfoSilva) [inline README.md:20]: We use the getter and setter of ES6, use this syntax:

```javascript
config.baseUrl = 'http://api.youraddress.local/v1';
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+  RestangularProvider.setDefaultHeaders({'Authorization': 'YOUR_DEFAULT_HEADERS'});
```
Comment (RodolfoSilva) [inline README.md:21]: We use the getter and setter of ES6, use this syntax:

```javascript
config.defaultHeaders = { 'Authorization': 'YOUR_DEFAULT_HEADERS' };
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
 # Ng-Restangular
 This project is the follow-up of the [Restangular](https://github.com/mgonto/restangular/) to Angular 2 written with Typescript.
+
+# Using it to your project
+Altough in beta, this project already will be able to be used in any other Angular 2 project. To do so, follow these steps:
+* <b>PS:</b> Ng-Restangular depends of Angular 2. You must install it before following the among steps.
+* Clone this repository (Cloning it inside your actual repository is recommended)
```
Comment (RodolfoSilva) [inline README.md:7]: Use `npm install rs-restangular --save`

**Comment by vaporwavie** [comment]
I just messed up with my fork, gonna redo it and then, I'll open a PR again :D

**Pull request closed without merging**